zha/application/platforms/fan/__init__.py

```diff
@@ -375,28 +373,15 @@ def async_update(self, _: Any = None) -> None:
         self.maybe_emit_state_changed_event()
 
 
-IKEA_SPEED_RANGE = (1, 10)  # off is not included
-IKEA_PRESET_MODES_TO_NAME = {
```

Comment:
Do these really not exist? I don't have the device so I can't check for myself.

Reply:
According to https://developers.home-assistant.io/docs/core/entity/fan 

> Preset modes should not include named (manual) speed settings as these should be represented as percentages.

These modes are basically just aliases for speeds.
The only real mode is auto.